plugin/manager.go

```diff
@@ -58,7 +58,7 @@ func Init(root string, remote libcontainerd.Remote, rs registry.Service, liveRes
 	root = filepath.Join(root, "plugins")
 	manager = &Manager{
 		libRoot:           root,
-		runRoot:           "/run/docker",
+		runRoot:           "/run/docker/plugins",
```

Comment:
For a separate PR, but this should not be hard-coded to `/run/docker` but instead use the daemon `exec root`